src/EventGrid/EventGrid/help/Get-AzEventGridDomain.md

```diff
@@ -0,0 +1,229 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.EventGrid.dll-Help.xml
+Module Name: Az.EventGrid
+online version: https://docs.microsoft.com/en-us/powershell/module/az.eventgrid/get-azeventgriddomain
+schema: 2.0.0
+---
+
+# Get-AzEventGridDomain
+
+## SYNOPSIS
+Gets the details of an Event Grid domain, or gets a list of all Event Grid domains in the current Azure subscription.
+
+## SYNTAX
+
+### ResourceGroupNameParameterSet (Default)
+```
+Get-AzEventGridDomain [[-ResourceGroupName] <String>] [-ODataQuery <String>] [-Top <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### DomainNameParameterSet
+```
+Get-AzEventGridDomain [-ResourceGroupName] <String> [-Name] <String> [-ODataQuery <String>] [-Top <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdEventSubscriptionParameterSet
+```
+Get-AzEventGridDomain [-ResourceId] <String> [-ODataQuery <String>] [-Top <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### NextLinkParameterSet
+```
+Get-AzEventGridDomain [-NextLink <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzEventGridDomain cmdlet gets either the details of a specified Event Grid domain, or a list of all Event Grid domains in the current Azure subscription.
+If the domain name is provided, the details of a single Event Grid domain is returned.
+If the domain name is not provided, a list of domains is returned. The number of elements returned in this list is controlled by the Top parameter. If the Top value is not specified or $null, the list will contain all the domains items. Otherwise, Top will indicate the maximum number of elements to be returned in the list.
+If more domains are still available, the value in NextLink should be used in the next call to get the next page of domains.
+Finally, ODataQuery parameter is used to perform filtering for the search results. The filtering query follows OData syntax using the Name property only. The supported operations include: CONTAINS, eq (for equal), ne (for not equal), AND, OR and NOT.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzEventGridDomain -ResourceGroup MyResourceGroupName -Name Domain1
```

Comment:
@ahamad-MS for new examples, please make sure that the corresponding output that the user would see from running the command is provided in the example code block

Reply:
Added output to the new examples/commands md files.